lesson11/index.js

```diff
@@ -0,0 +1,66 @@
+const ul = document.getElementById('js-lists');
+
+const fetchData = async() => {
+    const response = await fetch('https://jsondata.okiba.me/v1/json/KN8s0211027113740');
+    const obj = await response.json();
+    const listData = obj.data;
+
+    displayLoadingImage ();
+
+    await new Promise(resolve => {
+        setTimeout(() => {
+            resolve(listData);
+        }, 3000);
+    });
+    createList(listData);
+}
```

Comment:
IMO
The function name is "fetchData", but since we are also doing "createList", let's simply return the fetched data here. 
```suggestion
const fetchData = async() => {
    displayLoadingImage ();
    const response = await fetch('https://jsondata.okiba.me/v1/json/KN8s0211027113740');
    const obj = await response.json();
    return obj;
}
```
Then you should use try-catch here. It makes it easier to see where the error occurred.
We used to wait 3 seconds to get the pseudo-data, but from this issue, we don't need to use setTimeout.

Reply:
Thank you for your reviewing!
I did too many things in the fetchData function.
I fixed -> e5c45d0

Reply:
[I commented on it here!](https://github.com/kanakonagiri/js-lessons/pull/11#discussion_r739767848)